articles/protocols/oidc/index.md

```diff
@@ -29,7 +29,7 @@ Let's use the example we mentioned earlier, signing into Auth0 using your Google
 
 ## Access Tokens
 
-[Access Tokens](/tokens/access-token) are credentials that can be used by an application to access an API. Access Tokens can be an opaque string, JWT, or non-JWT token. Its purpose is to inform the API that the bearer of this token has been granted delegated access to the API and request specific actions (as specified by the scopes that have been granted).
+[Access Tokens](/tokens/access-token) are credentials that can be used by an application to access an API. Access Tokens can be an opaque string, JWT, or non-JWT token. Its purpose is to inform the API that the bearer of this token has been granted delegated access to the API and request specific actions (as specified by the [scopes](/scopes/current/oidc-scopes) that have been granted).
```

Comment:
OIDC scopes refer to the ID Token, where as the Access Token is an OAuth thing. You may want a different doc here (and/or want to move this to the ID Tokens section)?